src/txpool/getters.yaml

```diff
@@ -0,0 +1,69 @@
+- name: txpool_transactions
+  summary: Returns a list of all transactions that match the supplied filter conditions that are either pending for inclusion in the next block(s) or scheduled for future execution.
+  params:
+    - name: Filter
+      schema:
+        title: txpool filter
+        type: object
+        properties:
+          from:
+            title: from address
+            $ref: '#/components/schemas/address'
+          to:
+            title: to address
+            $ref: '#/components/schemas/address'
+          gas:
+            title: gas limit
+            $ref: '#/components/schemas/FilterOperators'
+          gasPrice:
+            title: gas price
+            $ref: '#/components/schemas/FilterOperators'
+          maxFeePerGas:
+            title: max fee
+            $ref: '#/components/schemas/FilterOperators'
+          maxPriorityFeePerGas:
+            title: max priority fee
+            $ref: '#/components/schemas/FilterOperators'
+          value:
+            title: value
+            $ref: '#/components/schemas/FilterOperators'
+          nonce:
+            title: nonce
+            $ref: '#/components/schemas/FilterOperators'
+  result:
+    name: Result
+    schema:
+      title: Result
+      schema:
+        $ref: '#/components/schemas/TxPoolTransactions'
+        title: transactions
+        type: object
+        properties:
+          pending:
+            title: pending
```

Comment:
If not yet present elsewhere, I think we should define which criteria make a tx pending and queue, to avoid interpretations that could results in different implementations

Reply:
Do you know if this is specified somewhere already? 
From my understanding pending means it's ready to be included in the next block
Queued is for the ones which are planned for future processing.

Reply:
I am thinking about the distinction of the two sets, so what makes a tx ready to be included or not. 
At first it seems easier to find the reasons why a tx in pool is not ready for inclusion, and the main reasons are:

- a previous tx is missing for the sender (nonce gap)
- not enough funds to pay for the gas

at the moment I only see these two, but I could miss something.